src/ng/httpBackend.js

```diff
@@ -109,7 +109,12 @@ function createHttpBackend($browser, createXhr, $browserDefer, callbacks, rawDoc
         }
       }
 
-      xhr.send(post || null);
+      if (post === false) {
```

Comment:
@johnhoffman, this build has failed because of a style error: `else` should be on the same line as the closing `}`:

```js
// WRONG
}
else {

// RIGHT
} else {
```

---
BTW, not all Travis failures means there is a problem with the code (you have to see the details; sometimes it's an unrelated timeout or something similar).
Additionally, you shouldn't create new PRs just to rerun the tests on Travis. Amending the latest commit and force-pushing to your repo, will update the PR and restart the Travis job.

Just so you know for future PRs :)
In any case, this is a legitimate failure, but I still wonder why only allow `false` ?

---
> I've been unable to build/test on my local machine per these instructions

What was the problem ? You should be able to run the tests and it is always better to have them pass locally first, in order to avoid losing time (Travis takes much longer) and wasting resources.